transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/WsnConnectorTest.java

```diff
@@ -335,12 +335,8 @@ public void doMessageReceived(final IoSessionEx session, Object message)
                 IoSessionEx sessionEx = session;
                 System.out.println("Acceptor: received message: " + Utils.asString(buf.buf()));
                 IoBufferAllocatorEx<?> allocator = sessionEx.getBufferAllocator();
-                session.write(allocator.wrap(asByteBuffer("Reply from acceptor"))).addListener(new IoFutureListener<IoFuture>() {
-                    @Override
-                    public void operationComplete(IoFuture future) {
-                        session.close(true);
-                    }
-                });
+                WriteFuture writeFuture = session.write(allocator.wrap(asByteBuffer("Reply from acceptor")));
+                writeFuture.addListener(future -> session.close(false));
```

Comment:
Originally session.close(true) which abruptly tears the connection. session.close(false) initiates orderly shutdown and test passes. Even though test passes, I am not sure why tearing down the connection doesn't fire wsn's ConnectFuture. Hence I am still investigating (as I cannot produce on my machine I am adding more diagnostic messages)